Add tests for Global Retention policy in hours

DHaussermann opened the pull request:
Add tests for Global Retention policy in hours

DHaussermann commented:
@saturninoabril I still have this marked as draft because there are tests in the pipeline failing. 
Can you please provide guidance on what's happening here?
![image](https://github.com/user-attachments/assets/4c35d839-1769-4bbd-a623-866a471ce70f)

I'm not even sure if this is in reference to the 2 files I added as part of my PR

saturninoabril commented on data/test-cases/suite/system-console/compliance/data-retention-policy/SetGlobalPolicyForAttachmentInHours.md:
```diff
@@ -0,0 +1,56 @@
+---
+# (Required) Ensure all values are filled up
+name: "Set global policy for messages in hours"
+status: Approved
```
```suggestion
status: Active
```

saturninoabril commented on data/test-cases/suite/system-console/compliance/data-retention-policy/SetGlobalPolicyForAttachmentInHours.md:
```diff
@@ -0,0 +1,56 @@
+mmctl: N/A
+playwright: N/A
+rainforest: []
+manual_test_environments: []
```
```suggestion
manual_test_environments:
```

saturninoabril commented on data/test-cases/suite/system-console/compliance/data-retention-policy/SetGlobalPolicyForMessagesInHours.md:
```diff
@@ -0,0 +1,54 @@
+---
+# (Required) Ensure all values are filled up
+name: "Set global policy for messages in hours"
+status: Approved
```
```suggestion
status: Active
```

saturninoabril commented on data/test-cases/suite/system-console/compliance/data-retention-policy/SetGlobalPolicyForMessagesInHours.md:
```diff
@@ -0,0 +1,54 @@
+mmctl: N/A
+playwright: N/A
+rainforest: []
+manual_test_environments: []
```
```suggestion
manual_test_environments:
```

saturninoabril reviewed (changes requested):
There's an error validating the changes. I suggested the changes and should passed on validation.

Locally, you may run `deno task validate` and see it returns `All changes are valid.`

<img width="1359" alt="Screenshot 2024-09-13 at 10 17 55 AM" src="https://github.com/user-attachments/assets/3153445c-38a0-438b-ac04-da7695145b3c">

DHaussermann commented:
@saturninoabril yes I see. I missed the `deno task validate` step and went straight to `deno task check` my mistake. 

But, I'm not seeing how tis get's me any closer, the error on my local looks the exact same :)

![image](https://github.com/user-attachments/assets/7944d296-baf2-4734-b26a-c591d4e776ab)

Where is the problem? 0/5 looks like Deno (maybe through a API) is making calls to remote files and something unexpected is in the response. Is this correct or am I misunderstanding the issue? 

Yesterday using the Brew package manager I updated Deno. I'm not clear on what I should do next.

saturninoabril commented:
I think you're on track. Have you tried applying the change requests I commented? Once applied, you may try local validation again and see if it fixes the error you're seeing.

DHaussermann commented:
@saturninoabril Awesome thank man!
![image](https://github.com/user-attachments/assets/6c6ae4c7-6299-4e5c-875c-7083e8ea5b04)

Locally it's fine now so let's see. The pipeline should say the same.

DHaussermann commented:
Implemented changes so default status is now correct and removed empty `[]` causing issue as the array I was trying to provide was not empty.

This is now ready for review.

DHaussermann commented:
@saturninoabril can you please review once more when you have a chance?

saturninoabril reviewed (approved):
Thanks @DHaussermann, LGTM!

saturninoabril commented:
@DHaussermann Test cases added as MM-T5627 and MM-T5628.
